printing_auto_stock_picking/tests/test_printing_auto_stock.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# Copyright 2022 Michael Tietz (MT Software) <mtietz@mt-software.de>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+from unittest import mock
+
+from odoo.addons.printing_auto_base.tests.common import (
+    PrintingPrinter,
+    TestPrintingAutoCommon,
+    print_document,
+)
+
+
+def exception(*args, **kwargs):
+    return
+
+
+@mock.patch.object(PrintingPrinter, "print_document", print_document)
+@mock.patch.object(logging.Logger, "exception", exception)
```

Comment:
why do you mock the exception? If you expect an exception to be logged it should be tested w/ `assertLogs`

Reply:
I think this should not be needed at all, since print_document is mocked.